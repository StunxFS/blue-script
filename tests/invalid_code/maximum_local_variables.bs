fn main() {
    var local0 = 0;
    var local1 = 1;
    var local2 = 2;
    var local3 = 3;
    var local4 = 4;
    var local5 = 5;
    var local6 = 6;
    var local7 = 7;
    var local8 = 8;
    var local9 = 9;
    var local10 = 10;
    var local11 = 11;
    var local12 = 12;
    var local13 = 13;
    var local14 = 14;
    var local15 = 15;
    var local16 = 16;
    var local17 = 17;
    var local18 = 18;
    var local19 = 19;
    var local20 = 20;
    var local21 = 21;
    var local22 = 22;
    var local23 = 23;
    var local24 = 24;
    var local25 = 25;
    var local26 = 26;
    var local27 = 27;
    var local28 = 28;
    var local29 = 29;
    var local30 = 30;
    var local31 = 31;
    var local32 = 32;
    var local33 = 33;
    var local34 = 34;
    var local35 = 35;
    var local36 = 36;
    var local37 = 37;
    var local38 = 38;
    var local39 = 39;
    var local40 = 40;
    var local41 = 41;
    var local42 = 42;
    var local43 = 43;
    var local44 = 44;
    var local45 = 45;
    var local46 = 46;
    var local47 = 47;
    var local48 = 48;
    var local49 = 49;
    var local50 = 50;
    var local51 = 51;
    var local52 = 52;
    var local53 = 53;
    var local54 = 54;
    var local55 = 55;
    var local56 = 56;
    var local57 = 57;
    var local58 = 58;
    var local59 = 59;
    var local60 = 60;
    var local61 = 61;
    var local62 = 62;
    var local63 = 63;
    var local64 = 64;
    var local65 = 65;
    var local66 = 66;
    var local67 = 67;
    var local68 = 68;
    var local69 = 69;
    var local70 = 70;
    var local71 = 71;
    var local72 = 72;
    var local73 = 73;
    var local74 = 74;
    var local75 = 75;
    var local76 = 76;
    var local77 = 77;
    var local78 = 78;
    var local79 = 79;
    var local80 = 80;
    var local81 = 81;
    var local82 = 82;
    var local83 = 83;
    var local84 = 84;
    var local85 = 85;
    var local86 = 86;
    var local87 = 87;
    var local88 = 88;
    var local89 = 89;
    var local90 = 90;
    var local91 = 91;
    var local92 = 92;
    var local93 = 93;
    var local94 = 94;
    var local95 = 95;
    var local96 = 96;
    var local97 = 97;
    var local98 = 98;
    var local99 = 99;
    var local100 = 100;
    var local101 = 101;
    var local102 = 102;
    var local103 = 103;
    var local104 = 104;
    var local105 = 105;
    var local106 = 106;
    var local107 = 107;
    var local108 = 108;
    var local109 = 109;
    var local110 = 110;
    var local111 = 111;
    var local112 = 112;
    var local113 = 113;
    var local114 = 114;
    var local115 = 115;
    var local116 = 116;
    var local117 = 117;
    var local118 = 118;
    var local119 = 119;
    var local120 = 120;
    var local121 = 121;
    var local122 = 122;
    var local123 = 123;
    var local124 = 124;
    var local125 = 125;
    var local126 = 126;
    var local127 = 127;
    var local128 = 128;
    var local129 = 129;
    var local130 = 130;
    var local131 = 131;
    var local132 = 132;
    var local133 = 133;
    var local134 = 134;
    var local135 = 135;
    var local136 = 136;
    var local137 = 137;
    var local138 = 138;
    var local139 = 139;
    var local140 = 140;
    var local141 = 141;
    var local142 = 142;
    var local143 = 143;
    var local144 = 144;
    var local145 = 145;
    var local146 = 146;
    var local147 = 147;
    var local148 = 148;
    var local149 = 149;
    var local150 = 150;
    var local151 = 151;
    var local152 = 152;
    var local153 = 153;
    var local154 = 154;
    var local155 = 155;
    var local156 = 156;
    var local157 = 157;
    var local158 = 158;
    var local159 = 159;
    var local160 = 160;
    var local161 = 161;
    var local162 = 162;
    var local163 = 163;
    var local164 = 164;
    var local165 = 165;
    var local166 = 166;
    var local167 = 167;
    var local168 = 168;
    var local169 = 169;
    var local170 = 170;
    var local171 = 171;
    var local172 = 172;
    var local173 = 173;
    var local174 = 174;
    var local175 = 175;
    var local176 = 176;
    var local177 = 177;
    var local178 = 178;
    var local179 = 179;
    var local180 = 180;
    var local181 = 181;
    var local182 = 182;
    var local183 = 183;
    var local184 = 184;
    var local185 = 185;
    var local186 = 186;
    var local187 = 187;
    var local188 = 188;
    var local189 = 189;
    var local190 = 190;
    var local191 = 191;
    var local192 = 192;
    var local193 = 193;
    var local194 = 194;
    var local195 = 195;
    var local196 = 196;
    var local197 = 197;
    var local198 = 198;
    var local199 = 199;
    var local200 = 200;
}
