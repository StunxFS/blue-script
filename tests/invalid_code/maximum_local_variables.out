tests/invalid_code/maximum_local_variables.bs:1:1: error: function `main` exceeded the maximum number of local variables allowed (200)
